### ISSUE TITLE
I'm reading the new tutorials, yay! (sorry for the issue title)

### ISSUE THREAD

**Issue by villares** [hx2A/py5book#50]
First of all a million thanks to @marcovicci for the incredible work with the tutorials!

A few questions/suggestions you might want to consider, after a very quick browsing:
 
- At http://py5dev.ixora.io/tutorials/intro_to_py5_and_python.html
   >  "or coding using an addon like the py5 mode for the Thonny IDE,"
   
   Could we add a link to https://github.com/tabreturn/thonny-py5mode ?

- Can we control the order of appearance on the index?

   - The first entry after the intro is the 3D one and the text seems to indicate we should see it further down.
![image](https://user-images.githubusercontent.com/3694604/205689960-297d94c6-2bb7-467d-9a59-02b0412e3fbc.png)

   - This issue reminded me that the original Processing tutorial have a "levels", *Beginner*, *Intermediate* and *Advanced*. Could we perhaps use these categories to organize the suggested reading order?

- In the conditionals tutorial, I think the character thing is a remnant of Java types, maybe we could simplify in Python... 

   - > "There are many simple types of data in py5 (and other programming languages). A single character of text is a character, many characters of text form a *string*, "

     So maybe we could have:
        > "There are many simple types of data in py5 (and other programming languages). Text, made of characters, is called a *string*, "

- The section titles, do we need the "in py5" part in the titles? Also, some wild suggestions you should ignore.
    - functions in py5 -> defining functions
    - three-dimensional graphics in py5 -> drawing 3D graphics with P3D
    - mouse and keyboard interaction in py5 -> mouse and keyboard interaction
    - ... complex custom shapes
    - ... strings and text

- On the other hand, maybe:
   - the *dictionary* data structure
   - the *list* data structure

- The engine task
     I think this should be right on top! 
![image](https://user-images.githubusercontent.com/3694604/205709857-d7706bd9-9860-405d-b70f-aae8c8dc59f1.png)

 
I'll add other comments as I read more, I hope...

**Comment by villares** [comment]
I see the tutorial order issue is fixed already! Great!

**Comment by hx2A** [comment]
@marcovicci the last PR looks good, we are almost there!

I noticed a problem with these two nearly identical pages:

http://py5dev.ixora.io/tutorials/intro_to_py5_and_python.html
http://py5dev.ixora.io/tutorials/intro_to_py5_and_python_01.html

The first one starts the chapter and the second one is the first subsection in the chapter. You added the Thonny link to the second one but it should have been added to the first one. Also, the second page should be removed.
